### ISSUE TITLE
老师您好，关于chapter5_RNN/time-series里面时间序列预测代码的问题

### ISSUE THREAD

**Issue by wmathor** [SherlockLiao/code-of-learn-deep-learning-with-pytorch#40]
第四个代码块"数据预处理"部分，最后一行，标准化不应该是(x-min_value)/(scalar)吗，为什么您的代码里没有减min_value

**Comment by ligeng0197** [comment]
确实, 老师这里没有减. 刚好我也看到这里了. 我觉得没有减的话放缩范围就变成了[min/scalar,1+min/scalar], 原LSTM输出隐藏状态h受tanh函数影响取值范围为[-1,1], 但是例子中最后有线性层将LSTM输出放大, 所以即使没有减也不太会影响性能.
